core/src/main/scala/org/apache/spark/api/r/SerDe.scala

```diff
@@ -166,6 +166,24 @@ private[spark] object SerDe {
     }
   }
 
+  def readEnv(in: DataInputStream): java.util.Map[Object, Object] = {
```

Comment:
My comment was only about the name for the function. It doesn't matter actually as its just an internal function.